test/concurrify.test.js

```diff
@@ -95,7 +95,7 @@ describe ('concurrify', function() {
       expect (actual).to.be.an.instanceof (ConcurrentIdentity);
     });
 
-    it ('reports being a ConcurrentIdentity from the same vendor and vendor', function() {
+    it ('reports being a ConcurrentIdentity from the same vendor', function() {
       var m = ConcurrentIdentity (Z.of (Identity, 1));
       expect (type (m)).to.equal ('my/ConcurrentIdentity@1');
```

Comment:
This test fails now, because `getTypeIdentifier (Identity)` is returning `"Identity"` now, as opposed to `"my/Identity@1"`.